storage/integration_test.go

```diff
@@ -744,6 +746,127 @@ func TestIntegration_ObjectsRangeReader(t *testing.T) {
 	}
 }
 
+func TestIntegration_ObjectReadGRPC(t *testing.T) {
+	// TODO: Remove this once v2 rollout is complete.
+	t.SkipNow()
+
+	ctx := context.Background()
+	client, err := newClientWithGRPC(ctx)
+	if err != nil {
+		t.Error(err)
+	}
+	defer client.Close()
+	bkt := client.Bucket(grpcBucket)
+
+	objName := uidSpace.New()
+	obj := bkt.Object(objName)
+	contents := []byte("Hello, world this is a grpc request")
+
+	if err := retry(ctx, func() error {
+		w := obj.NewWriter(ctx)
+		if _, err := w.Write(contents); err != nil {
+			return fmt.Errorf("Failed to write contents: %v", err)
+		}
+		if err := w.Close(); err != nil {
+			return fmt.Errorf("Failed to close writer: %v", err)
+		}
+		return nil
+	}, nil); err != nil {
+		t.Fatal(err)
+	}
+	defer func() {
+		if err := obj.Delete(ctx); err != nil {
+			log.Printf("failed to delete test object: %v", err)
+		}
+	}()
+
+	r, err := obj.NewReader(ctx)
+	if err != nil {
+		t.Error(err)
+	}
+	defer r.Close()
+	buf := make([]byte, len(contents))
+	n, err := r.Read(buf)
```

Comment:
Should probably just use io.Copy on the reader.

Reply:
Done!